### ISSUE TITLE
UA Ultra (Unifi Access Ultra)

### ISSUE THREAD

**Issue by ojan29** [hjdhjd/homebridge-unifi-access#16]
### Homebridge UniFi Access Version

v1.3.0

### Homebridge Platform and OS

Docker Synology (Ubuntu Jammy (22.04.4 LTS))

### Homebridge Version

v1.8.3

### Node Version

v20.15.0

### UniFi OS Version

v4.0.6

### UniFi Access Controller Version

v2.1.2

### Describe the problem

Hello,
My UA Ultra is not supported...

### Homebridge HBUA JSON configuration

```json
[7/16/2024, 4:26:30 PM] [UniFi Access] Initializing UniFi Access platform...
[7/16/2024, 4:26:30 PM] [UniFi Access] No host or IP address has been configured.
[7/16/2024, 4:26:30 PM] Loading 1 accessories...
[7/16/2024, 4:28:12 PM] [UniFi Access] Initializing UniFi Access platform...
[7/16/2024, 4:28:12 PM] [UniFi Access] Initializing child bridge **:**:**:**:**
[7/16/2024, 4:28:12 PM] Loading 1 accessories...
[7/16/2024, 4:28:13 PM] [UniFi Access] Launched child bridge with PID 28807
[7/16/2024, 4:28:14 PM] [UniFi Access] Loaded homebridge-unifi-access v1.3.0 child bridge successfully
[7/16/2024, 4:28:14 PM] Registering platform 'homebridge-unifi-protect.UniFi Protect'
[7/16/2024, 4:28:14 PM] [UniFi Access] No host or IP address has been configured.
[7/16/2024, 4:28:14 PM] Loaded 5 cached accessories from cachedAccessories.******.
[7/16/2024, 4:28:14 PM] Homebridge v1.8.0 (HAP v0.12.0) (UniFi Access) is running on port ****.
[7/16/2024, 4:31:58 PM] [UniFi Access] Initializing UniFi Access platform...
[7/16/2024, 4:31:58 PM] [UniFi Access] Initializing child bridge **:**:**:**:**
[7/16/2024, 4:31:58 PM] Loading 1 accessories...
[7/16/2024, 4:31:59 PM] [UniFi Access] Launched child bridge with PID 30119
[7/16/2024, 4:31:59 PM] [UniFi Access] Loaded homebridge-unifi-access v1.3.0 child bridge successfully
[7/16/2024, 4:31:59 PM] Loaded 0 cached accessories from cachedAccessories.*******.
[7/16/2024, 4:31:59 PM] [UniFi Access] No host or IP address has been configured.
[7/16/2024, 4:31:59 PM] Homebridge v1.8.0 (HAP v0.12.0) (UniFi Access) is running on port ****.
[7/16/2024, 4:31:59 PM] Registering platform 'homebridge-unifi-protect.UniFi Protect'
[7/16/2024, 4:48:28 PM] [UniFi Access] Restarting child bridge...
[7/16/2024, 4:48:28 PM] Got SIGTERM, shutting down child bridge process...
[7/16/2024, 4:48:33 PM] [UniFi Access] Child bridge process ended
[7/16/2024, 4:48:33 PM] [UniFi Access] Process Ended. Code: 143, Signal: null
[7/16/2024, 4:48:40 PM] [UniFi Access] Restarting Process...
[7/16/2024, 4:48:43 PM] [UniFi Access] Launched child bridge with PID 31057
[7/16/2024, 4:48:46 PM] Registering platform 'homebridge-unifi-access.UniFi Access'
[7/16/2024, 4:48:46 PM] [UniFi Access] Loaded homebridge-unifi-access v1.3.0 child bridge successfully
[7/16/2024, 4:48:46 PM] Loaded 0 cached accessories from cachedAccessories.******.
[7/16/2024, 4:48:47 PM] Homebridge v1.8.0 (HAP v0.12.0) (UniFi Access) is running on port ****.
(node:31057) [DEP0123] DeprecationWarning: Setting the TLS ServerName to an IP address is not permitted by RFC 6066. This will be ignored in a future version.
(Use `node --trace-deprecation ...` to show where the warning was created)
[7/16/2024, 4:48:49 PM] [UniFi Access] UCKP: Connected to 192.168.1.210 (UniFi Access v2.1.2+6107 running on UniFi OS 4.0.6).
[7/16/2024, 4:48:49 PM] [UniFi Access] UCKP: Discovered UA Ultra: UA Ultra 4edb [UA Ultra] (address: 192.168.1.87 mac: ****).
[7/16/2024, 4:48:49 PM] [UniFi Access] UCKP: UniFi Access device type 'UA-ULTRA' is not currently supported, ignoring: UA Ultra 4edb [UA Ultra].
[7/16/2024, 4:57:28 PM] [UniFi Access] Initializing UniFi Access platform...
[7/16/2024, 4:57:28 PM] [UniFi Access] Initializing child bridge **:**
[7/16/2024, 4:57:29 PM] Loading 1 accessories...
[7/16/2024, 4:57:29 PM] [UniFi Access] Launched child bridge with PID 31865
[7/16/2024, 4:57:30 PM] [UniFi Access] Loaded homebridge-unifi-access v1.3.0 child bridge successfully
[7/16/2024, 4:57:30 PM] Loaded 0 cached accessories from cachedAccessories.****.
[7/16/2024, 4:57:30 PM] Registering platform 'homebridge-unifi-protect.UniFi Protect'
[7/16/2024, 4:57:32 PM] [UniFi Access] UCKP: Connected to 192.168.1.210 (UniFi Access v2.1.2+6107 running on UniFi OS 4.0.6).
[7/16/2024, 4:57:32 PM] [UniFi Access] UCKP: Discovered UA Ultra: UA Ultra 4edb [UA Ultra] (address: 192.168.1.87 mac: ****).
[7/16/2024, 4:57:32 PM] [UniFi Access] UCKP: UniFi Access device type 'UA-ULTRA' is not currently supported, ignoring: UA Ultra 4edb [UA Ultra].
```

### Relevant log output

```shell
[7/16/2024, 4:57:32 PM] [UniFi Access] UCKP: Connected to 192.168.1.210 (UniFi Access v2.1.2+**** running on UniFi OS 4.0.6).
[7/16/2024, 4:57:32 PM] [UniFi Access] UCKP: Discovered UA Ultra: UA Ultra 4edb [UA Ultra] (address: 192.168.1.87 mac: ).
[7/16/2024, 4:57:32 PM] [UniFi Access] UCKP: UniFi Access device type 'UA-ULTRA' is not currently supported, ignoring: UA Ultra 4edb [UA Ultra].
```

### Acknowledgment that you are only running UniFi OS and UniFi Access releases from the Ubiquiti Official release channel

- [X] I acknowledge that I am not running any early access / beta firmwares or operating systems from either Ubiquiti or Apple.

**Comment by hjdhjd** [comment]
Already addressed in another issue. Please read through them before opening new issues. This will be addressed in a future release.

**Comment by hjdhjd** [comment]
Addressed in v1.4.0.
